src/index.ts

```diff
@@ -55,6 +87,14 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 
 			await emit('apiImpl', path.join(outputPath, relativeSourceOutputPath, 'impl', `${context.generator().toIdentifier(group.name)}.ts`), 
 				{ ...rootContext, ...group, ...doc }, false, hbs)
+
+			for (const operation of operations) {
+				if (operation.requestBody?.defaultContent.mediaType.mediaType === 'multipart/form-data') {
```

Comment:
Maybe we can pull the code out of `hasOperationSupportingMultipart` into a function and re-use it here. It feels like we should have one way of determining this.